step-ca.yaml

```diff
@@ -1,17 +1,17 @@
 package:
   name: step-ca
   version: "0.28.2"
-  epoch: 4
+  epoch: 5
   description: A private certificate authority (X.509 & SSH) & ACME server for secure automated certificate management, so you can use TLS everywhere & SSO for SSH.
   copyright:
     - license: Apache-2.0
 
 environment:
   contents:
     packages:
-      - busybox
-      - ca-certificates-bundle
-      - go
+      - pcsc-lite-dev
+      - pkgconf-dev
+      - step
```

Comment:
we can consider omitting all this, go toolchain coming via go/build is enough here (build and tests are passing locally). Is there a reason to include these build time deps here? 💭

Reply:
Weird, build was failing on me without having those deps, also step-ca requires step package as a build time dep.

Reply:
I just omit `pkgconf-dev`, thanks!